[USING] Wallet Load (#704)
zkSNACKs/WasabiDoc: docs/using-wasabi/WalletLoad.md

---
{
  "title": "Wallet Load",
  "description": "A detailed guide about loading multiple wallets in Wasabi. This is the Wasabi documentation, an archive of knowledge about the open-source, non-custodial and privacy-focused Bitcoin wallet for desktop."
}
---

# Wallet Load

There are two ways of loading your wallets in Wasabi, and you can load multiple wallets at the same time.

[[toc]]

---

## Wallet Explorer

> On the right side of Wasabi is the `Wallet Explorer`, where you see an alphabetically sorted list of all the previously [generated](/WalletGeneration.md) wallets.

Reviewer: ```suggestion
On the right side of Wasabi is the `Wallet Explorer`, where you see an alphabetically sorted list of all the previously [generated](/using-wasabi/WalletGeneration.md) wallets.
```

![](/WalletExplorerUnloaded.png)

> You can load a wallet simply by double clicking the line.

Reviewer 2: ```suggestion
You can load a wallet simply by double clicking on it or by right-clicking on it and then choosing `Load Wallet`.
```

> A coin icon will appear to indicate the loaded wallets.

Reviewer 2: ```suggestion
A coin will appear on the wallet icon to indicate the loaded wallets.
```

Author: Yes, this is in the PR #702
